src/Agent.Worker/Build/TfsVCSourceProvider.cs

```diff
@@ -679,15 +679,15 @@ public static class WorkspaceUtil
                         bool expectedCloak = definitionMapping.MappingType == DefinitionMappingType.Cloak;
                         if (tfMapping.Cloak != expectedCloak)
                         {
-                            executionContext.Debug($"Expected mapping[{i}] cloak: '{expectedCloak}'. Actual: '{tfMapping.Cloak}'");
+                            executionContext.Output($"Expected mapping[{i}] cloak: '{expectedCloak}'. Actual: '{tfMapping.Cloak}'");
```

Comment:
you might want to localize all the strings, since it's now go to `output` instead of `debug`

Reply:
`StringUtil.Loc()`

Reply:
Good point, I didn't think of that.